### PR TITLE
[issue 21] - Enabling WildFly Helm Charts based provisioning test

### PR DESCRIPTION
**Pull request by fabiobrz**
## Description

Enabling WildflyHelmChartProvisionerTest

Part of #21 

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [x] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x] Code contributed by me aligns with current project license: [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0)
 - [x] Pull Request contains a description of the changes
 - [x] Pull Request does not include fixes for multiple issues/topics
 - [x] Code is self-descriptive and/or documented
 - [ ] I have implemented unit tests to cover my changes
 - [x] I tested my code in OpenShift

### CODE REVIEW

File: .ci/openshift-ci/build-root/e2e-test.sh [hunk 1]
```diff
@@ -16,33 +16,71 @@ printenv KUBEADMIN_PASSWORD_FILE
...
-# both for xtf.openshift.admin.* and xtf.openshift.master.* properties
+export TEST_CLUSTER_URL=$(oc whoami --show-server)
+
 export SYSADMIN_USERNAME=kubeadmin
 export SYSADMIN_PASSWORD=$(cat "${KUBEADMIN_PASSWORD_FILE}")
-export TEST_CLUSTER_URL=$(oc whoami --show-server)
-oc login "${TEST_CLUSTER_URL}" -u "${SYSADMIN_USERNAME}" -p "${SYSADMIN_PASSWORD}" --insecure-skip-tls-verify=true
-export SYSADMIN_TOKEN=$(oc whoami -t)
+
+# note that for some reason it takes a few minutes for this to be loaded by OCP and authentication start working
```
Comment (tommaso-borgato) [inline .ci/openshift-ci/build-root/e2e-test.sh:24]: @fabiobrz why do we need yet another cluster admin?

File: global-test.properties [hunk 2]
```diff
@@ -19,8 +19,13 @@ intersmash.wildfly.operators.package_manifest=wildfly
...
 
 # Wildfly images
-intersmash.wildfly.image=quay.io/wildfly/wildfly-s2i-jdk17
-intersmash.wildfly.runtime.image=quay.io/wildfly/wildfly-runtime-jdk17
+intersmash.wildfly.image=quay.io/wildfly/wildfly-s2i-jdk17:latest
+intersmash.wildfly.runtime.image=quay.io/wildfly/wildfly-runtime-jdk17:latest
+
+# Wildfly Helm Charts
+intersmash.wildfly.helm.charts.repo=https://github.com/wildfly/wildfly-charts.git
+intersmash.wildfly.helm.charts.branch=main
```
Comment (tommaso-borgato) [inline global-test.properties:27]: @fabiobrz just upstream latest? why not some tag for stability's sake?

File: .ci/openshift-ci/build-root/e2e-test.sh (hunk 1 above)
Comment (fabiobrz) [reply .ci/openshift-ci/build-root/e2e-test.sh:24]: See https://github.com/Intersmash/intersmash/issues/48 :slightly_smiling_face:

File: global-test.properties (hunk 2 above)
Comment (fabiobrz) [reply global-test.properties:27]: You might be right. I'll ask

**Pull request merged**